idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -207,25 +182,16 @@ def build(self):
         self.mass_flow_coeff = Var(self.flowsheet().config.time,
                                    initialize=0.0735,
```

Comment:
Just an observation, but these are rather specific values for a general purpose model.